deep_quoridor/src/quoridor_env.py

```diff
@@ -156,6 +156,7 @@ def _get_observation(self, agent_id):
         walls = self.walls.copy()
 
         return {
+            "my_turn": self.agent_selection == agent_id,
             "board": board,
             "walls": walls,
             "my_walls_remaining": self.walls_remaining[agent_id],
```

Comment:
nit: would you mind also updating (or deleting) the large comment at the top of the file which explains the observation state format, so that it's not out of sync with the implementation? I'm afraid otherwise it will lead to confusion.